tests/generator/test_passthrough.py

```diff
@@ -122,22 +122,22 @@ def test_passthrough_type_nm_devices(self):
       networkmanager:
         passthrough:
           connection.uuid: 87749f1d-334f-40b2-98d4-55db58965f5f
-          connection.type: dummy''')
```

Comment:
We should keep upstream's `dummy` type here: https://developer-old.gnome.org/NetworkManager/stable/settings-dummy.html